### ISSUE TITLE
Update extension icon 

### ISSUE THREAD

**Issue by kellyhutchins** [Esri/arcgis-js-vscode-snippets#41]
Currently we use a default esri icon.  This issue is to identify and replace it with a more meaningful icon. Once a new icon has been identified it can be updated using the icon option in package. json

**Comment by kimmykokonut** [comment]
Is the goal to have it be one of Esri's icons/logos or to design something specific?  
If it's ArcGIS, maybe that logo? 
![image](https://github.com/user-attachments/assets/b77b8dd7-8485-4a16-8e3b-6ac9f207cef6)
Or the SDK logo on the microsoft store for their ArcGIS Maps SDK for .NET Samples:

**Comment by hhkaos** [comment]
Hi Kim!

Good question.
* The [first one](https://private-user-images.githubusercontent.com/55254544/377623947-b77b8dd7-8485-4a16-8e3b-6ac9f207cef6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkyNjA2NzUsIm5iZiI6MTcyOTI2MDM3NSwicGF0aCI6Ii81NTI1NDU0NC8zNzc2MjM5NDctYjc3YjhkZDctODQ4NS00YTE2LThlM2ItNmFjOWYyMDdjZWY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE4VDE0MDYxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE5ZWQ0YzY4YzNjOWJjNzY0MzMzZTJhMjBkZjgxYTdlYTQ3NGE0OWVhYTc3M2Y3NDVlMjc4Mzc2YzU4ZWVkZjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.3L3lc-JKb9xiOzo_tSxZjwu6wi8O5ZTNS7wKZk8otlE). actually is not a valid Esri logo as [I stated here](https://commons.wikimedia.org/wiki/File_talk:ArcGIS_logo.png) 😄 . 
* The second one is the right one. Attached at high resolution below.
* The [third one](https://private-user-images.githubusercontent.com/55254544/377624421-38b43a67-35b3-4973-b5f9-63f42e1a15e6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkyNjA2NzUsIm5iZiI6MTcyOTI2MDM3NSwicGF0aCI6Ii81NTI1NDU0NC8zNzc2MjQ0MjEtMzhiNDNhNjctMzViMy00OTczLWI1ZjktNjNmNDJlMWExNWU2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE4VDE0MDYxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMxMDVjYjg3NzM0MDU5OTljOWIzZDFkYTQwYmRmNGMxOTU4ZjMyNWNjMGM1YWUyMWU2MDY1MzdmZmRkZDQ5MTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.K1myIQVuwYdbuPFu5ED06h4ndTaWBP164fw4MLNvWjU) is the one used for the ArcGIS Maps SDK for **Native** SDKs :).

Thanks!
![arcgis-maps-sdk-for-javascript-256 (1)](https://github.com/user-attachments/assets/9159b0fb-9273-4fbe-b0c3-bbf5dc08c271)

**Comment by kimmykokonut** [comment]
Thank you for the clarification.  So the desire is the blue JS icon? If so, I can take this on. I wasn't sure if there was a plan to design a new one, which I do not feel qualified to do outside of Esri.

**Comment by hhkaos** [comment]
Right, the desired one is this:
![](https://private-user-images.githubusercontent.com/826965/377906059-9159b0fb-9273-4fbe-b0c3-bbf5dc08c271.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk0MzMzMjQsIm5iZiI6MTcyOTQzMzAyNCwicGF0aCI6Ii84MjY5NjUvMzc3OTA2MDU5LTkxNTliMGZiLTkyNzMtNGZiZS1iMGMzLWJiZjVkYzA4YzI3MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAyMFQxNDAzNDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05Nzk5NzMyOWFjNmZjNDY5ZGQwNzZiMjZjOTUxYjcyYjUyNTZlZTFiZTU4OWM0MDM4YTU2ZThlNjI3NTk5Mzk5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.UCGAn6D2UE1Yi1BZnS9BCDcHUpL0LiWJCNMTHlzGj-U)

So please, feel free to start working on it.
Thanks! 😄

**Comment by hhkaos** [comment]
hi @kimmykokonut, let me know if you need any help with this 🙂

**Comment by nooras** [comment]
Hi @hhkaos 
Is it open to work on? Shall i raise a PR for this? Can u please let me know on which page icon needs to update?

**Comment by hhkaos** [comment]
yes it is!, the icon is referenced in https://github.com/Esri/arcgis-js-vscode-snippets/blob/master/package.json#L20 . So, it should be as simple as uploading [this icon](https://github.com/Esri/arcgis-js-vscode-snippets/issues/41#issuecomment-2422615079) to the repo and make sure the package.json is pointing to it 😄

**Comment by nooras** [comment]
Hi @hhkaos 
how can I verify in local that newly added image is working fine?
